postman/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "Postman",
+  "maintainer": "integrations-partnerships@postman.com",
+  "manifest_version": "1.0.0",
+  "name": "postman",
+  "metric_prefix": "PostmanMonitor_",
+  "metric_to_check": "",
+  "creates_events": true,
+  "short_description": "Integration to observe metric from Postman's Monitor run in Datadog",
```

Comment:
metric --> metrics

I also think you can delete the word "run"

Reply:
Updated